docs/application-author.md

```diff
@@ -270,14 +270,14 @@ COCKROACHDB_CONF_PORT=8090
 Here's how the mount paths look like:
 
 ```
-var
-├── data
-│   ├── COCKROACHDB_CLUSTERIP
-│   ├── COCKROACHDB_CONF_PORT
+
+/bindings
+├── COCKROACHDB_CLUSTERIP
+├── COCKROACHDB_CONF_PORT
 
 ```
 
-Instead of `/var/data`, you can specify a custom binding root path by specifying the same in `spec.mountPathPrefix`, example,
```

Comment:
Document volume mount. Explain the behavior of these attributes `bindAsFiles` and `mountPath` and the special env. var (`SERVICE_BINDING_ROOT`).

Reply:
Added a commit